python/tvm/tir/analysis/analysis.py

```diff
@@ -331,3 +331,32 @@ def OOBChecker():
         The result pass
     """
     return _ffi_api.OOBChecker()  # type: ignore
+
+
+def communicative_deep_equal(lhs: PrimExpr, rhs: PrimExpr) -> bool:
+    """Deeply compare two nested expressions that have communicative equality.
+
+    Parameters
+    ----------
+    lhs : PrimExpr
+        The left operand.
+
+    rhs : PrimExpr
+        The right operand.
+
+    Returns
+    -------
+    result : bool
+        The comparison result
+
+    Note
+    ----
+
+    This function is an extension of py:func:`tvm.ir.expr_deep_equal`, it can
+    handle commutativity. The function will not return true for (x + y) vs (y + x).
```

Comment:
Thanks for the review！The description here is indeed misleading, I have replaced "The function" with the exact function name to avoid misunderstanding.